pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
+
+- Search the working directory recursively for files containing a regex pattern:
+
+`ugrep "{{pattern}}"`
+
+- Search for a case-[i]nsensitive regex pattern:
+
+`ugrep -i "{{pattern}}"`
```

Comment:
```suggestion
- Search for a case-insensitive regex pattern:

`ugrep --ignore-case "{{pattern}}"`
```

Reply:
I understand, but please see navarroaxel comments and our discussion, which I think was resolved in keeping the short options. There are several good reasons for this: 1) grep also uses the same short options so using long options loses the relationship of ugrep to grep, 2) the same short options are used by other tools like `ag`, 3) certain short options are very common for file search tools such as `-i`, `-l`, `-r` (for recursion), and `-z`, 4) using these long options requires more mental effort by the reader and a lot more typing to enter the command and options that should be the same as the good old grep tool but now look different. Isn't tldr supposed to be as short and concise as possible? As a computer scientist, I know that all short options must have long option equivalents as strongly recommended by standards. But that does not mean we should always use them, when common well-knowns short options are easy to remember and use.

Reply:
Maybe this is personally my problem but for me it's hard to remember what `-i`|`-l`|`-r` mean because not always there is any rules how these short options are written. The worth case when there are several options with just different case. When I read my scripts with such options I need go to tldr/man page/google it to understand meaning. Long options easier to read at my glance (but not always).

Reply:
I'm not saying that all short options are always good and I agree generally with your assessment on the readability argument. However, for grep search tools the options `-i`, `-l` and `-z` are well understood and used without people thinking much about it. Also, as another example of this, `-r` is typically recursion, e.g. `cp`, `mv`, `rm`, `ls` and many other utilities use this option too. So these are more or less "natural" for a lack of better way to say this.

Reply:
Oh, I see now after reviewing more pages that tldr grep page also uses long options. But that's awful IMHO, very sorry to say... The `cp`, `rm` etc. utilities have pages with short options simply because they never had long options to begin with. OK, so I agree that for consistency long options should be used with tldr pages.

Reply:
We are thinking about modify Tl;Dr specification to allow user switch between pages with short/long options. If you have smth to say please leave your comment.